src/roman_datamodels/stnode.py

```diff
@@ -204,7 +213,7 @@ def convert_val(val):
             return dict((key, convert_val(val)) for (key, val) in self.items())
         else:
             return dict((key, convert_val(val)) for (key, val) in self.items()
-                        if not isinstance(val, (np.ndarray, NDArrayType)))
+                        if not isinstance(val, np.ndarray))
```

Comment:
`NDArrayType` is not an instance of `np.ndarray`, so we might still need this.  When reading an ASDF file all the arrays come back as NDArrayType.